logging/src/jvmMain/kotlin/io/spine/logging/JvmLogger.kt

```diff
@@ -84,6 +84,11 @@ private class ApiImpl(private val delegate: FluentLogger.Api): JvmLogger.Api {
         return this
     }
 
+    override fun every(n: Int): JvmLogger.Api {
+        delegate.every(n)
```

Comment:
@yevhenii-nadtochii @alexander-yevsyukov 
I think _if_ our final goal is to remove Flogger altogether, maybe we should consider re-implementing this feature by ourselves? WDYT.